src/plays/typingtestgame/components/styles.css

```diff
@@ -0,0 +1,104 @@
+@import url("https://fonts.googleapis.com/css?family=Muli&display=swap");
```

Comment:
@neelesh133 Do not override tag selectors this way. Please read this

https://docs.reactplay.io/How-To-Guides/how-to-style-play

You need add a unique class name of ur choice and use that as a prefix for your styles.

Reply:
I will surely resolve it in a while. :)

Reply:
Unfortunately, the project has somehow stopped working and giving me an error on "npm start". I have installed node_modules through "npm i --legacy-peer-deps" successfully. The screenshot of the error is below-

![image](https://github.com/reactplay/react-play/assets/119878088/fe5db415-537a-48b4-b488-659381297ac7)

Can you please help me out there ?

Reply:
Are you still facing an issue?